runtime/src/chpl-visual-debug.c

```diff
@@ -66,6 +66,9 @@ int chpl_vdebug = 0;
 
 #define TID_STRING(buff, tid) (chpl_task_idToString(buff, CHPL_TASK_ID_STRING_MAX_LEN, tid))
 
+#define CHPL_VDEBUG_GETPUT_FORMAT_NAMES "tv srcNodeID dstNodeID commTaskID addr raddr elemSize typeIndex length commID lineNumber fileno"
+#define CHPL_VDEBUG_GETPUT_FORMAT_STRING(name) #name ": %lld.%06ld %d %d %s %#lx %#lx %zd %d %zd %d %d %d\n"
```

Comment:
This is a good change, but you can do without the leading "CHPL_" because it's not a global symbol.  Also, it might be a bit easier to read if you just did something like:

    #define VDEBUG_GETPUT_FORMAT_STRING "%s: %lld.%06ld %d %d %s %#lx %#lx %zd %d %zd %d %d %d\n"

and then in usage, something like:

    printf(VDEBUG_GETPUT_FORMAT_STRING, "nb_put", ...